doc/examples.md

```diff
@@ -1,38 +1,69 @@
+Every measurement folder also needs to be enriched with metadata, which itself is specified with another [JSON schema](../src/main/resources/schemas/ont-metadata.schema.json).
 
-```JAVA
-import life.qbic.datamodel.instruments.*;
+The final map contains an additional `metadata` property for each measurement, that for example can look like this:
 
-public class SchemaImport {
-  public static void main(String[] args) {
-    InputStream inputStream = OxfordNanoporeInstrumentOutput.getSchemaAsStream();
-  }
+```
+{
+    "name": "QABCD001AB_E12A345a01_PAE12345",
+    "path": "./",
+    "children": [
+        {
+            "name": "20200122_1217_1-A1-B1-PAE12345_1234567a",
+            "metadata":  {
+                "adapter": "flongle",
+                "asic_temp": "32.631687",
+                "base_caller": "Guppy",
+                "base_caller_version": "3.2.8+bd67289",
+                "device_type" : "promethion",
+                "flow_cell_id": "PAE26306",
+                "flow_cell_product_code": "FLO-PRO002",
+                "flow_cell_position": "2-A3-D3",
+                "hostname": "PCT0094",
+                "protocol": "sequencing/sequencing_PRO002_DNA:FLO-PRO002:SQK-LSK109:True",
+                "started": "2020-02-11T15:52:10.465982+01:00"
+            },
+            "path": "./20200122_1217_1-A1-B1-PAE12345_1234567a",
+            ...
+            ]
 }
 ```
 
-**Nf Core pipeline output**
+The [data model API](../src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeExperiment.groovy) outlines how to create an instance of `OxfordNanoporeExperiment`.
 
-The schema for the nf core pipeline output can be accessed via the
-`NfCorePipelineResult` class:
 
-```JAVA
-import life.qbic.datamodel.instruments.*;
+###Nf-Core dataset
 
-public class SchemaImport {
-  public static void main(String[] args) {
-    InputStream inputStream = PipelineOutput.getSchemaAsStream();
-  }
-}
-```
\ No newline at end of file
+In order to create an instance of type `NfCorePipelineResult`, you need to provide a map that provides content following the [Nfcore Pipeline Output Schema JSON](./src/main/resources/schemas/bioinformatics-analysis-result-set.schema.json).  
+
+The [data model API](../src/main/groovy/life/qbic/datamodel/datasets/NfCorePipelineResult.groovy) outlines how to create an instance of `NfCorePipelineResult`.
+
+## Validate data structure
+
+Each object contains a schema within which can be used to test if a parsed file tree conforms to the expected structure.  
+The validation itself is performed via the `SchemaLoader` class provided by the [Everit](https://github.com/everit-org/json-schema) package.
+
+## Implementation examples 
+
+The Nanopore experiment object is implemented in the [core utils API](https://github.com/qbicsoftware/core-utils-lib/blob/master/src/main/groovy/life/qbic/utils/NanoporeParser.groovy). 
```

Comment:
Whats the difference of the Nanopore Experiment here in DML and in CUL? It seems as if its a  duplicate